eva-server/src/main/java/uk/ac/ebi/eva/server/ws/ga4gh/GA4GHBeaconWSServer.java

```diff
@@ -56,6 +61,8 @@ public GA4GHBeaconResponse beacon(@RequestParam("referenceName") String chromoso
                                       @RequestParam("start") long start,
                                       @RequestParam("allele") String allele,
                                       @RequestParam("datasetIds") List<String> studies,
+                                      @RequestParam(name = "contigNamingConvention", required = false)
+                                      ContigNamingConvention contigNamingConvention,
```

Comment:
Did we decide to change the GA4GH endpoints as well? Asking because I'm just not sure what the spec allows...

Reply:
The specs allow to have more parameters but not mandatory.
In any case we're not compliant with the spec already and the specs have moved on to Beacon v2.